R/common-column-row-expectations.R

```diff
@@ -95,4 +95,28 @@ returnColsWithMoreThanFiftyCategories <- function(df) {
     }
   }
   colList
-}
\ No newline at end of file
+}
+
+#' @title
+#' Check if a data frame only has numeric columns.
+#' @description Check if a dataframe only has numeric columns
+#' @param df A dataframe
+#' @return A boolean
+#' 
+#' @export
+#' @references \url{http://healthcare.ai}
+#' @seealso \code{\link{healthcareai}}
+#' @examples 
+#' df <- data.frame(a=c(1,2,3),
+#'                  b=c(NA,3,2))
+#' isNumeric(df)
+isNumeric <- function(df) {
+  a <- sapply(df, is.numeric)
```

Comment:
Thanks for putting this in common and using roxygen! Would you mind adding a couple of unit tests for this?